lib/registration.pm

```diff
@@ -96,6 +96,8 @@ sub fill_in_registration_data {
             }
             send_key 'alt-n';    # next, all addons selected
             for my $addon (split(/,/, get_var('SCC_ADDONS', ''))) {
+                # most modules don't have license, skip them
+                next unless ($addon =~ /ha|geo|sdk|we|live|\brt\b|lgm|wsm/);
```

Comment:
I would say regexps hit a limit here. The $addon variable is e.g. 'ha', no?

Then IMO it's easier to read as 
```
next unless grep { $addon eq $_ } qw(ha geo sdk we live rt lgm wsm);
```

(even though I would prefer some comment about the abbrevations used - some are clearer than others :)